web/controllers/screeningController/index.ts

```diff
@@ -301,3 +303,106 @@ export async function updateScreenerByMailHandler(
         next();
     }
 }
+
+/**
+ * @api {POST} /courses getCourses
+ * @apiVersion 1.0.1
+ * @apiDescription
+ * 
+ * Retrieves the first 20 courses that match the specified filters.
+ * 
+ * 
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName getCourses
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X POST -H "Token: <AUTHTOKEN>" https://dashboard.corona-school.de/api/screening/courses
+ *
+ * @apiParam (JSON Body) {string=} courseState the course state ("created", "submitted", "allowed", "denied", "cancelled")
+ * @apiParam (JSON Body) {string=} search A query text to be searched in the title and description
+ */
+export async function getCourses(req: Request, res: Response) {
+    try {
+        const { courseState, search } = req.body;
```

Comment:
Since this should be a GET request, you should use query parameters instead of a request body.

Reply:
Well actually this was rather by design (https://dropbox.tech/developers/limitations-of-the-get-method-in-http, https://evertpot.com/dropbox-post-api/) ... as the other APIs are also not quite RESTful i thought I could be lazy and use a JSON body instead of doing urlencoding. But I'm fine changing that if you prefer GET.